lib/Runtime/Base/JnDirectFields.h

```diff
@@ -181,6 +181,7 @@ ENTRY(decodeURI)
 ENTRY(decodeURIComponent)
 ENTRY2(default_, _u("default"))
 ENTRY2(delete_, _u("delete")) // "delete" cannot be an identifier in C++ so using "delete_" instead
+ENTRY2(star, _u("*"))
```

Comment:
Should we indicate this is a symbol (change name to "_star" or similar)? It seems unlikely, but a built-in API might come along with the name of star (String.prototype.star maybe).